src/main/resources/templates/fragments/inputs/checkbox.html

```diff
@@ -25,7 +25,8 @@
              th:checked="${checked}"
              th:attr="
               aria-invalid=${hasError},
-              data-follow-up=${followUpId}">
+              data-follow-up=${followUpId},
+              onchange=|toggleVisibility(this, '${toggleVisibility}');|">
```

Comment:
Let's make this more generic 
```suggestion
              onchange=${onChange}">
```